src/plugins/intel_cpu/tests/functional/shared_tests_instances/single_layer_tests/generate_proposals.cpp

```diff
@@ -232,4 +233,28 @@ INSTANTIATE_TEST_SUITE_P(
         ::testing::Values(ov::element::Type_t::i32),
         ::testing::Values(CommonTestUtils::DEVICE_CPU)),
     GenerateProposalsLayerTest::getTestCaseName);
+
+struct GenerateProposalsBenchmarkTest : ov::test::BenchmarkLayerTest<GenerateProposalsLayerTest> {};
+
+TEST_P(GenerateProposalsBenchmarkTest, DISABLED_GenerateProposals_Benchmark) {
+    SKIP_IF_CURRENT_TEST_IS_DISABLED()
+    run_benchmark("GenerateProposals", std::chrono::milliseconds(2000), 10000);
+}
+
+INSTANTIATE_TEST_SUITE_P(
+        smoke_BasicTest,
```

Comment:
Please, exclude such tests from pre-commit
remove *smoke* prefix. Thanks!

Reply:
Those are marked as DISABLED at TEST_P macro, so are no enabled. @iefode please clarify if you want them to be completely removed from the codebase? It looks reasonable for me to leave them in some form just for the usage reference. If yes, may be you have a suggestion on the better place for them, or we may try to make suggestion if you wish.

Reply:
I mean that these tests should be out of pre-commit scope (is defined by using `smoke` prefix). it's applicable also for disabled tests

Reply:
Thanks for clarification! It would soon be addressed in PR update.